mypyc/genops.py

```diff
@@ -1709,14 +1712,14 @@ def load_outer_env(self, base: Value, outer_env: Environment, line: int) -> Valu
 
```

Comment:
Is adding stuff to the `class_ir` still needed here? (Come to think of it, was it ever actually needed?)
I think that this will result in adding attributes to the `ClassIR` for all of the variables in the parent, even though the ones not corresponding to free variables won't be used.

Unfortunately because of #136, this information is not actually exposed in genops tests.